modules/integration/tests/src/test/java/org/wso2/appmanager/integration/utils/APPMAdminDashboardRestClient.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.appmanager.integration.utils;
+
+import org.json.JSONObject;
+import org.wso2.appmanager.integration.utils.bean.BusinessOwner;
+import org.wso2.appmanager.integration.utils.bean.BusinessOwnerCreateRequest;
+import org.wso2.carbon.automation.test.utils.http.client.HttpRequestUtil;
+import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+public class APPMAdminDashboardRestClient {
+    private String backEndUrl;
+    private Map<String, String> requestHeaders = new HashMap<String, String>();
+
+    public APPMAdminDashboardRestClient(String backEndUrl) throws MalformedURLException {
+        this.backEndUrl = backEndUrl;
+        if (requestHeaders.get(AppmTestConstants.CONTENT_TYPE) == null) {
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/e263afed698e7dbd1253c0136563513fd4051eea